src/app/partials/header/header.component.html

```diff
@@ -0,0 +1,20 @@
+<header>
+  <h1>{{ title }}</h1>
+
+  <nav>
+    <ul>
+      <li><a routerLinkActive="active" [routerLink]="['/home']">Accueil</a></li>
+      <li><a href="">Meilleurs ventes</a></li>
+      <li><a href="">Electronics</a></li>
+      <li><a href="">Jewelery</a></li>
+      <li><a href="">Men's clothing</a></li>
+      <li><a href="">Women's clothing</a></li>
+      <li><a routerLinkActive="active" [routerLink]="['/products']">Tous les produits</a></li>
```

Comment:
Qu'est-ce que tu dirais de refacto ce code en créant un objet ItemMenu de tel sorte que tu puisses remplacer ce code par :
```suggestion
      <li *ngFor="let itemMenu of itemMenus"><a routerLinkActive="active" [routerLink]="[itemMenu.url]">{{itemMenu.label}</a></li>
```

Reply:
C'est prévu de faire un truc dynamique avec les catégories après avec le getAll des categories pour que pour chaque catégories il crée le lien dans le nav.
Toi, t'es plus rapide, tu prépares déjà le terrain directement. :)

Reply:
Oui, on fera dans une autre PR.